### DISCUSSION TITLE
E-mail notification from QField Cloud in case of error

### DISCUSSION THREAD

**Discussion by Churros396** [opengisch/QField#4600] [Q&A]
Hi, is it possible to set up E-Mail notifications in case of errors like for delta apply, jobs failing or the project status turning red. It would be good to counter the data loss or fix the project fast. I haven't discovered such an option yet.

**Comment by Sekou225** [comment]
Will be very helpfull

**Comment by ZsanettMed** [comment]
Hello @Churros396  and @Sekou225!

Currently, QFieldCloud does not support the feature you're asking for. However, we greatly appreciate your feedback as it aids us in enhancing and updating QFieldCloud. Your suggestion has been shared on our [QField Ideas Platform](https://ideas.qfield.org/qfieldcloud-feature-requests/p/email-notifications-for-errors) where you and others can vote for it. 

Best regards
Zsanett
